.gitignore

```diff
@@ -31,3 +31,4 @@ lerna-debug.log
 .vscode
 # For vim
 *.swp
+.yarn
```

Comment:
Why did the .gitignore get updated?

Also to fix the DashboardWrapper not being found you will need to also make the change from this PR https://github.com/patternfly/patternfly-react/pull/10191.

Reply:
@kmcfaul I've updated `.gitignore` due to yarn generate .yarn folder on my machine and want to ignore that folder changes.  Let me know if this change cause any issue 

I've fixed DashboardWrapper not found issue.